X10D/src/Core/IntrinsicUtility.cs

```diff
@@ -176,8 +295,8 @@ public static Vector128<float> HorizontalOr(Vector128<float> lhs, Vector128<floa
     }
 
     /// <summary>
-    ///     <para>Horizontally apply OR operation on adjacent pairs of 32-bit integer elements in lhs and rhs.</para>
-    ///     <para>Operation:</para>
+    ///     <br>Horizontally apply OR operation on adjacent pairs of 32-bit integer elements in lhs and rhs.</br>
+    ///     <br>Operation:</br>
```

Comment:
Why `<br>`? That's a single tag and it represents a new line character. Encapsulating lines in `<br>TEXT</br>` makes no sense. The same goes for lines 221-222, 261-262, and a lot of other method xmldocs in this file.

Reply:
Well I have old habit of doing that, I'll whip up a commit to fix them.